example/transaction.py

```diff
@@ -1,69 +1,45 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 
-import time
-
-import requests
-
-from chainlibpy import Transaction, Wallet
-from chainlibpy.amino import Coin, StdFee
-from chainlibpy.amino.message import MsgSend
+from chainlibpy.generated.cosmos.base.v1beta1.coin_pb2 import Coin
+from chainlibpy.grpc_client import GrpcClient
+from chainlibpy.wallet import Wallet
+
+# NOTE:
+# Recommend to use pystarport(https://pypi.org/project/pystarport/) to setup a testnet locally
+
+DENOM = "basecro"
+# Obtained from {directory_started_pystarport}/data/chainmaind/accounts.json
+# To recover one of the genesis account
+MNEMONIC_PHRASE = "rhythm armed olympic accident cost brief since right rail ramp abstract live modify pluck convince balance onion auction jewel mansion flock scare bleak sample"  # noqa 501
```

Comment:
how about using env variable?
instead of putting mnemonics directly?

Reply:
If privacy is a concern, I could put the mnemonic phrase as `first ... last`. An environment variable is an overkill for this, as it is just a simple example to demonstrate how to interact with the library. I will make a new commit and change all subsequent mnemonic phrases and addresses in the example and `README.md` to include `...` to hide the real ones.